.github/workflows/test.yml

```diff
@@ -14,8 +14,6 @@ jobs:
     steps:
       - name: 🛎️ Checkout
         uses: actions/checkout@v3
-        with:
-          ref: ${{ github.head_ref }}
```

Comment:
removing this allows checkout to work on forks.

Reply:
Nice catch, I've recently merge a PR that fixes it #49 . Could you please rebase?

Reply:
@FrancescoSaverioZuppichini - i tried rebasing with your change and it broke GHA workflows...

```
Fetching the repository
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/accept-JPG*:refs/remotes/origin/accept-JPG* +refs/tags/accept-JPG*:refs/tags/accept-JPG*
  The process '/usr/bin/git' failed with exit code 1
  Waiting 11 seconds before trying again
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/accept-JPG*:refs/remotes/origin/accept-JPG* +refs/tags/accept-JPG*:refs/tags/accept-JPG*
  The process '/usr/bin/git' failed with exit code 1
  Waiting 12 seconds before trying again
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/accept-JPG*:refs/remotes/origin/accept-JPG* +refs/tags/accept-JPG*:refs/tags/accept-JPG*
  Error: The process '/usr/bin/git' failed with exit code 1
  ```

it did seem to work when i deleted the `with:` block though.